### PR TITLE
Added tire pressure

### PR DESCRIPTION
**Pull request by NirKli**
Added new functionality to teslamate to record tire pressure via Tesla API. 
Tire pressure fields in the API was added by Tesla in 2022.4 version.
Added new panel in drive-internal dashboard to show the records of the tires. 
The recorded data is shown by Bar value and can be converted to PSI, by a new postgresql function
The result is shown in the picture below.

image

### CODE REVIEW

**Comment by tomS3210** [comment]
Thanks a lot ! Hope it will be included soon in the release. How can we test it ?

**Comment by NirKli** [comment]
> Thanks a lot ! Hope it will be included soon in the release. How can we test it ?
> 
> 

Sure, let me know if there is a problem.

File: lib/teslamate/vehicles/vehicle/summary.ex [hunk 1]
```diff
@@ -13,7 +13,7 @@ defmodule TeslaMate.Vehicles.Vehicle.Summary do
     odometer shift_state charge_port_door_open time_to_full_charge charger_phases
     charger_actual_current charger_voltage version update_available update_version is_user_present geofence
     model trim_badging exterior_color wheel_type spoiler_type trunk_open frunk_open elevation power
-    charge_current_request charge_current_request_max
+    charge_current_request charge_current_request_max tpms_pressure_fl tpms_pressure_fr tpms_pressure_rl tpms_pressure_rr
```
Comment (adriankumpf) [inline lib/teslamate/vehicles/vehicle/summary.ex:16]: Fields added here are automatically published via MQTT. Could you please add them to the documentation: `website/docs/integrations/mqtt.md`

File: priv/repo/migrations/20220617170400_add_tire_pressures.exs [hunk 2]
```diff
@@ -0,0 +1,35 @@
+defmodule TeslaMate.Repo.Migrations.AddTirePressures do
+  use Ecto.Migration
+
+  def change do
+    alter table(:positions) do
+      add_if_not_exists(:tpms_pressure_fl, :numeric, precision: 4, scale: 1)
+      add_if_not_exists(:tpms_pressure_fr, :numeric, precision: 4, scale: 1)
+      add_if_not_exists(:tpms_pressure_rl, :numeric, precision: 4, scale: 1)
+      add_if_not_exists(:tpms_pressure_rr, :numeric, precision: 4, scale: 1)
```
Comment (adriankumpf) [inline priv/repo/migrations/20220617170400_add_tire_pressures.exs:9]: Let's use the plain `add`s here. If a user's database already contains these columns, something is wrong.

```suggestion
      add(:tpms_pressure_fl, :numeric, precision: 4, scale: 1)
      add(:tpms_pressure_fr, :numeric, precision: 4, scale: 1)
      add(:tpms_pressure_rl, :numeric, precision: 4, scale: 1)
      add(:tpms_pressure_rr, :numeric, precision: 4, scale: 1)
```

File: priv/repo/migrations/20220617170400_add_tire_pressures.exs [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    end
+
+      execute("DROP FUNCTION IF EXISTS public.convert_tire_pressure(double precision, character varying);", &noop/0)
+
```
Comment (adriankumpf) [inline priv/repo/migrations/20220617170400_add_tire_pressures.exs:13]: This should not be necessary.

```suggestion
```

**Review by adriankumpf — CHANGES_REQUESTED** [review]

File: lib/teslamate/vehicles/vehicle/summary.ex (hunk 1 above)
Comment (NirKli) [reply lib/teslamate/vehicles/vehicle/summary.ex:16]: Done :)

**Review by adriankumpf — APPROVED** [review]
Brilliant! Thank you!

**Pull request merged**